sktime/performance_metrics/forecasting/probabilistic/_classes.py

```diff
@@ -85,8 +95,42 @@ def _evaluate_by_index(self, y_true, y_pred, multioutput, **kwargs):
             )
         return out_series
 
+    def _check_alpha(self, alpha):
+        if isinstance(alpha, list):
+            alpha = np.array(alpha)
+        if isinstance(alpha, np.ndarray):
+            self.multiple_alphas = True
+            if not all(((alpha > 0) & (alpha < 1))):
+                raise ValueError("Alpha must be between 0 and 1.")
+            return alpha
+        if isinstance(alpha, float):
+            self.multiple_alphas = False
+            if not ((alpha > 0) & (alpha < 1)):
+                raise ValueError("Alpha must be between 0 and 1.")
+            return [alpha]
+        raise TypeError("Alpha should be a float or numpy array")
 
-class PinballLoss(_BaseProbForecastingErrorMetric):
+    def _check_ys(self, y_true, y_pred, multioutput):
+        if multioutput is None:
+            multioutput = self.multioutput
+        _, y_true, y_pred, multioutput = _check_reg_targets(y_true, y_pred, multioutput)
```

Comment:
This check no longer works. It used to work as I would check it column by column rather than all at once, but this is inefficient and I've now made my own version which implements similar checks.